ui/pages/Project/components/RnaSeqOutlierVolcanoPlot.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { extent } from 'd3-array'
+import { axisBottom, axisLeft } from 'd3-axis'
+import { scaleLinear, scaleLog } from 'd3-scale'
+import { select } from 'd3-selection'
```

Comment:
This file is split from the `ui/pages/Project/components/RnaSeqOutliers.jsx` for sharing between expressions and splicing outlier data.